src/fundamentals.html

```diff
@@ -516,7 +516,7 @@ <h5 id="fund_defaults">Default values of attributes</h5>
    <h4 id="fund_globatt">Attributes Shared by all MathML Elements</h4>
 
    <p>In addition to the attributes described specifically for each element,
-   the attributes in the following table are allowed on every MathML element.
+   the HTML attributes in the following table are allowed on every MathML element.
```

Comment:
```suggestion
   the attributes in the following table are allowed on every MathML element.
```

I wouldn't add HTML here I think it's confusing. xref isn't HTML at all and MatHMl's href is a global attribute so not really like HTML/a's href which is just on a (and similar elements)